src/EventStore.Core/ClusterVNodeOptions.cs

```diff
@@ -727,26 +937,67 @@ public record InterfaceOptions {
 
 			internal static InterfaceOptions FromConfiguration(IConfigurationRoot configurationRoot) => new() {
 				GossipOnSingleNode = configurationRoot.GetValue<bool?>(nameof(GossipOnSingleNode)),
+#pragma warning disable 618
```

Comment:
You can disable the pragma warning before the constructor and then re-enable it at the end of the constructor rather than repeating it each time